lib/quantity/math.ex

```diff
@@ -154,4 +154,70 @@ defmodule Quantity.Math do
       :error -> raise(ArgumentError)
     end
   end
+
+  @doc """
+  Divide a Quantity by a scalar or another Quantity
+
+  iex> Quantity.div(~Q[15 $], ~Q[10 banana])
+  ~Q[1.5 $/banana]
+  iex> Quantity.div(~Q[15 $], ~d[10])
+  ~Q[1.5 $]
+  iex> Quantity.div(~Q[15 $], ~Q[10 $])
+  ~d[1.5]
```

Comment:
Nice. Perhaps add an empty line between tests for consistency with other doctests?

Reply:
Consistency is good :+1: 
Added in 5c5d447e2fdf85f82d30b107a783a40543047126

Reply:
Super. Looks good.